src/vm/object.h

```diff
@@ -1060,11 +1060,8 @@ typedef PTR_StringObject STRINGREF;
  *
  * Special String implementation for performance.   
  *
- *   m_ArrayLength  - Length of buffer (m_Characters) in number of WCHARs
- *   m_StringLength - Length of string in number of WCHARs, may be smaller
- *                    than the m_ArrayLength implying that there is extra
- *                    space at the end. The high two bits of this field are used
- *                    to indicate if the String has characters higher than 0x7F
+ *   m_StringLength - Length of string in number of WCHARs. The high two bits of this field
```

Comment:
`The high two bits of this field are used to indicate if the String has characters higher than 0x7F` is not true either. Could you please delete it as well?